<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QLCDNumber" name="lcdNumber_Latitude"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_Logintude">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Longitude&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_Latitude">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Latitude&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLCDNumber" name="lcdNumber_Longitude"/>
       </item>
       <item row="0" column="4">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_Camera">
            <property name="text">
             <string>Camera</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>GPS</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QMdiArea" name="mdiArea_3"/>
    </item>
    <item row="2" column="0">
     <widget class="QMdiArea" name="mdiArea_2"/>
    </item>
    <item row="3" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="HomeMenu">
    <property name="title">
     <string>Home</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionReports"/>
    <addaction name="actionAdvice"/>
    <addaction name="separator"/>
    <addaction name="actionCamera"/>
    <addaction name="actionLog_Off"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionInterface_Type"/>
    <addaction name="actionCom_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionData_Settings"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionVisualization"/>
    <addaction name="separator"/>
    <addaction name="actionTrend"/>
    <addaction name="separator"/>
    <addaction name="actionResults"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionRegister"/>
    <addaction name="actionEdit_Profile"/>
   </widget>
   <addaction name="HomeMenu"/>
   <addaction name="menuUser"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuData"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit the application</string>
   </property>
  </action>
  <action name="actionCom_Settings">
   <property name="text">
    <string>Com. Settings</string>
   </property>
   <property name="statusTip">
    <string>Communication Setting</string>
   </property>
  </action>
  <action name="actionData_Settings">
   <property name="text">
    <string>Data Settings</string>
   </property>
  </action>
  <action name="actionVisualization">
   <property name="text">
    <string>Visualization</string>
   </property>
  </action>
  <action name="actionTrend">
   <property name="text">
    <string>Trend</string>
   </property>
  </action>
  <action name="actionReports">
   <property name="text">
    <string>Reports</string>
   </property>
   <property name="statusTip">
    <string>Reports</string>
   </property>
  </action>
  <action name="actionAdvice">
   <property name="text">
    <string>Advice</string>
   </property>
   <property name="statusTip">
    <string>Alerts Window</string>
   </property>
  </action>
  <action name="actionLog_Off">
   <property name="text">
    <string>Log Off</string>
   </property>
   <property name="statusTip">
    <string>Log Off</string>
   </property>
  </action>
  <action name="actionRegister">
   <property name="text">
    <string>Register</string>
   </property>
   <property name="statusTip">
    <string>Edit your profile</string>
   </property>
  </action>
  <action name="actionEdit_Profile">
   <property name="text">
    <string>Edit Profile</string>
   </property>
  </action>
  <action name="actionResults">
   <property name="text">
    <string>Results</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
  <action name="actionInterface_Type">
   <property name="text">
    <string>Interface Type</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
